src/app/feature/job/pages/manage-jobs/manage-jobs.component.scss

```
@@ -0,0 +1,89 @@
+select {
+	background-image: none;
+}
+
+// Smooth modal animations
+.animate-in {
+	animation-duration: 0.3s;
+	animation-timing-function: var(--easing);
+	animation-fill-mode: both;
+}
+
+@keyframes fadeIn {
```

Stylelint (17.7.0)

[error] Expected keyframe name "fadeIn" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)